.github/workflows/gh-pages-deployment.yml

```diff
@@ -23,6 +22,13 @@ jobs:
     - name: Install Dependencies
       run: npm install
 
+    - name: Add subdomain
+      uses: richardrigutins/replace-in-files@v2
+      with:
+        files: 'package.json'
+        search-text: '"homepage": "/",'
+        replacement-text: '"homepage": "https://ai-cfia.github.io/nachet-frontend",'
```

Comment:
looking into this as this seems like an hacky solution

Reply:
the PUBLIC_URL did the job